core/foundation/inc/RStringView.h

```diff
@@ -47,7 +51,9 @@ namespace std {
 #ifndef R__HAS_STOD_STRING_VIEW
    inline double stod(std::string_view str, size_t *pos)
    {
-      return std::stod(str.to_string(),pos);
+// to_string() method didn't make into C++17, but the following should be compatible
+// with std::sting_view and std::experimental::string_view.
```

Comment:
That's a good comment for the change, but a bit out of context for the code. Just remove it, please.